### DISCUSSION TITLE
BLE device Integration

### DISCUSSION THREAD

**Discussion by Jens-Wymeersch** [mKeRix/room-assistant#847] [Q&A]
Did anyone integrate Tile, NUT or any other Bluetooth device with Room Assistant ? If so, would it be possible to outline how you got it to work?
I want to track if my cat is in the house or not.

**Comment by mKeRix** [comment]
I've seen a few posts for both the Tile and the NUT, personally I've also used RadBeacons successfully. You can use the GitHub search to browse the repository for more info, or alternatively check out the Home Assistant Community as people have also posted their setups there sometimes.

**Reply by Jens-Wymeersch** [reply]
So, after a few ups and downs, I was able to find the BLE mac address. Now what i've noticed is that it doesn't update that much the status of the app. Any suggestions how to mitigate this ? @mKeRix

**Reply by Jens-Wymeersch** [reply]
Just for the record, after a week testing and hearing from others, Tile can't really be used as it just doesn't send any Bluetooth information to the RPI. @mKeRix if you have different findings, please let me know.

**Reply by sharpopotamus** [reply]
You actually can use the Tile just fine, but only if you don't pair it with the Tile app (which does kinda defeat the point)

**Reply by Jens-Wymeersch** [reply]
> You actually can use the Tile just fine, but only if you don't pair it with the Tile app (which does kinda defeat the point)

Initially, i didn't pair the device but it never transmitted his location to the room assistant. Can you please explain what you did to overcome this ?

**Reply by lgbrownjr** [reply]
Just my two cents...
I've had mixed results in using tiles that are paired to the app.  Most work solid, but a couple have stop working with RA - i think they change mac addresses for one reason or the other.  I have a total of two mates, two wallet tiles, two pros, and one Plantronics voyager headset (with built-in tile).   Both pros work great, and i have them attached to my dog's collars.  One of the mates is working great too.  But the other mate stopped one day about a month ago - It works with the tile app, but room assistant no longer sees it.

When scanning for it, i noticed that the tile might have changed its mac address, but still when i change the mac address in room assistant to match, it still doesn't find the tile.  Similarly, the headset acts the same, but when i find its new mac address, RA picks it up and resumes tracking.  Again, this change in mac address never affects the the pairing with tile - that part of the tiles functionality is never affected.

I forgot to mention, that i also have a fitbit with a built-in tile, and so far It woks with both the tile app and RA without issues 
To be clear, in all cases the tiles are paired to the app.
